files/stat.py

```diff
@@ -331,6 +347,37 @@
 from ansible.module_utils._text import to_bytes
 
 
+def format_attributes(attributes):
+    ATTRIBUTES = {
+        'A': 'noatime',
+        'a': 'append',
+        'c': 'compressed',
+        'C': 'nocow',
+        'd': 'nodump',
+        'D': 'dirsync',
+        'e': 'extents',
+        'E': 'encrypted',
+        'h': 'blocksize',
+        'i': 'immutable',
+        'I': 'indexed',
+        'j': 'journalled',
+        'N': 'inline',
+        's': 'zero',
+        'S': 'synchronous',
+        't': 'notail',
+        'T': 'blockroot',
+        'u': 'undelete',
+        'X': 'compressedraw',
+        'Z': 'compresseddirty',
+    }
```

Comment:
style nit, not a blocker.  If we move this list to a global then it keeps code and data separate  Can make the code's flow more readable.  Also means the data is only instantiated once but that's not a concern here.